### PR TITLE
[USING] explain bip 125

### PR DESCRIPTION
**Pull request by MaxHillebrand**
this ready for review branch adds explanation for bip 125 replace by fee signaling.

### CODE REVIEW

File: docs/using-wasabi/BIPs.md [hunk 1]
```diff
@@ -75,7 +75,9 @@ On the TestNet and on the RegTest Wasabi deviates from the standard and uses `m/
 :::details
 ### BIP 125: Opt-In full Replace-by-Fee Signaling
 
-[BIP 125: Opt-In full Replace-by-Fee Signaling](https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki) is activated for a subset of transactions chosen randomly, so to decrease wallet fingerprinting.
+[BIP 125: Opt-In full Replace-by-Fee Signaling](https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki) is a policy that allows the sender to signal that the transaction can be replaced in the future.
+This is useful in order to bump the fee of an unconfirmed transaction to incentivise miners for faster confirmation.
```
Comment (yahiheb) [inline docs/using-wasabi/BIPs.md:79]:
```suggestion
This is useful in order to bump the fee of an unconfirmed transaction to incentivize miners to include the transaction in their candidate block for faster confirmation.
```

File: docs/using-wasabi/BIPs.md [hunk 2]
```diff
@@ -75,7 +75,9 @@ On the TestNet and on the RegTest Wasabi deviates from the standard and uses `m/
 :::details
 ### BIP 125: Opt-In full Replace-by-Fee Signaling
 
-[BIP 125: Opt-In full Replace-by-Fee Signaling](https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki) is activated for a subset of transactions chosen randomly, so to decrease wallet fingerprinting.
+[BIP 125: Opt-In full Replace-by-Fee Signaling](https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki) is a policy that allows the sender to signal that the transaction can be replaced in the future.
```
[the remark this answers is not in the document]
Comment (MaxHillebrand) [reply docs/using-wasabi/BIPs.md:78]: not sure... we usually say `sender`. though `spender` works too, I just think it's less common...

[the remark this answers is not in the document]
Comment (yahiheb) [reply docs/using-wasabi/BIPs.md:78]: https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki#abstract
I took a look at the abstract of the BIP and there spender is used.

Comment (lontivero) [inline docs/using-wasabi/BIPs.md:78]: This creates more confusion. What does it mean that the transaction can be replaced in the future?

**Pull request closed without merging**